src/llm_vm/onsite_llm.py

```diff
@@ -461,8 +465,8 @@ def generate(self,prompt, max_length=100,**kwargs): # both tokenizer and model t
         return ans['choices'][0]['message']['content']
 
     def finetune(self, dataset, optimizer, c_id):
-        print("fine tuning isn't supported by OpenAI on this model", file=sys.stderr)
-        exit()
+        message = "fine tuning isn't supported by OpenAI on this model"
```

Comment:
After that #210 pr should be fixed that, but i don't have to opportunity to test it,
OpenAI allow the finetune with this specific model `gpt-3.5-turbo-0613`